src/Template/Common/ComposerJsonTemplate.php

```diff
@@ -43,8 +43,15 @@ public function __toString(): string
                     "emulgeator/enum": "^1.0",
                     "emulgeator/array-to-class-mapper": "^0.1",
                     "psr/http-message": "^1.0",
+                    "psr/http-message-implementation": "^1.0",
```

Comment:
Without the -implementation part when used as a dependency, it will not force the parent project to actually include an implementation for the PSR. The base PSR only includes the interface, but no implementations.